target/linux/qualcommax/ipq807x/base-files/lib/upgrade/platform.sh

```diff
@@ -66,6 +66,21 @@ platform_do_upgrade() {
 		fw_setenv upgrade_available 1
 		nand_do_upgrade "$1"
 		;;
+	linksys,mx4200v1|\
+	linksys,mx4200v2)
```

Comment:
You should keep it in alphabetical order.

Reply:
The resolution causes a merge conflict